resources/js/views/AdminDashboardView.vue

```diff
@@ -1,21 +1,316 @@
 @extends('layouts.app')
 <template>
```

Comment:
There is no need to import the api module twice in the script section.
It's a good practice to declare the props and data sections above the mounted and methods sections.